src/linters/ArcanistESLintLinter.php

```diff
@@ -23,7 +23,7 @@ public function getInfoDescription() {
     }
 
     public function getVersion() {
-        $output = exec('eslint --version');
+        list($output) = execx('%C --version', $this->getExecutableCommand());
 
         if (strpos($output, 'command not found') !== false) {
```

Comment:
should we improve on this by checking the success case (with a regex match) instead or is this good enough?